pkg/operator/observe_config.go

```diff
@@ -64,27 +65,30 @@ func (c ConfigObserver) sync() error {
 		return err
 	}
 	observedConfig := map[string]interface{}{}
-	etcdURLs := []string{}
 
 	// we read the etcd endpoints from the endpoints object and then manually pull out the hostnames to get the etcd urls for our config.
 	// Setting them observed config causes the normal reconciliation loop to run
+	etcdURLs := []string{}
 	etcdEndpoints, err := c.kubeClient.CoreV1().Endpoints(etcdNamespaceName).Get("etcd", metav1.GetOptions{})
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
 	if etcdEndpoints != nil {
 		for _, subset := range etcdEndpoints.Subsets {
 			for _, address := range subset.Addresses {
-				etcdURLs = append(etcdURLs, "https://"+address.Hostname+":2379")
+				etcdURLs = append(etcdURLs, "https://"+address.Hostname+"."+etcdEndpoints.Annotations["alpha.installer.openshift.io/dns-suffix"]+":2379")
```

Comment:
same comment as for the other repo about the dot

Reply:
> same comment as for the other repo about the dot

I think we fail in general.  If this lasts longer than a few weeks we're going to have some pretty significant issues in general.